.github/workflows/tests.yaml

```diff
@@ -70,10 +71,11 @@ jobs:
               - bindings/prql-dotnet/**
               - bindings/prql-lib/**
               - .github/workflows/test-dotnet.yaml
-            devcontainer:
+            devcontainer-push:
               - .devcontainer/**/*Dockerfile
-              - .github/workflows/build-devcontainer.yaml
               - Taskfile.yml
+            devcontainer-test:
+              - .github/workflows/build-devcontainer.yaml
```

Comment:
Sorry, I missed the need to include the Dockerfile here. I will fix it later.

Reply:
Sorry! Good spot. Lmk if you prefer I fix this...

Reply:
Thank you. I'm glad you are doing this as I am about to go to work and can't do this right away (just note that it takes 3+ hours to get through the CI as the cache doesn't exist until after it is pushed from the main branch)